Rakefile

```diff
@@ -1,167 +1,127 @@
-    unless $osx_success
-      puts "** Build failed"
-      exit(-1)
-    end
-    $osx_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests_OSX' -destination 'platform=OS X' test")
-    puts "\033[0;31m! OS X unit tests failed" unless $osx_success
-    if $osx_success
-        puts "** All tests executed successfully"
-    else
-        exit(-1)
-    end
+  test(REPLICATION_ACCEPTANCE_WS, REPLICATION_ACCEPTANCE_ENCRYPTED_OSX, OSX_DEST)
 end
 
 desc "Run the replication acceptance tests for iOS with encrypted datastores"
 task :encryptionreplicationacceptanceios do
-    # build using xcpretty as otherwise it's very verbose when running tests
-    $ios_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 4S' build | xcpretty; exit ${PIPESTATUS[0]}")
-    unless $ios_success
-      puts "** Build failed"
-      exit(-1)
-    end
-    $ios_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 4S' test")
-    puts "\033[0;31m! iOS unit tests failed" unless $ios_success
-    if $ios_success
-        puts "** All tests executed successfully"
-    else
-        exit(-1)
-    end
+  test(REPLICATION_ACCEPTANCE_WS, REPLICATION_ACCEPTANCE_ENCRYPTED_IOS, IOS_DEST)
 end
 
-desc "pod update all included projects"
-task :podupdate => [:podupdatetests] do
-  sh "for i in ReplicationAcceptance Project\ndo\ncd $i ; pod update ; cd ..\ndone"
-end
+#
+#  Update docs
+#
 
 desc "Build docs and install to Xcode"
 task :docs do
   system("appledoc --keep-intermediate-files --project-name CDTDatastore --project-company Cloudant -o build/docs --company-id com.cloudant -i Classes/vendor -i Classes/common/touchdb Classes/")
 end
+
+#
+#  Helper methods and defines
+#
+
+# Workspaces
+CDTDATASTORE_WS = 'CDTDatastore.xcworkspace'
+ENCRYPTION_WS = 'EncryptionTests/EncryptionTests.xcworkspace'
+REPLICATION_ACCEPTANCE_WS = './ReplicationAcceptance/ReplicationAcceptance.xcworkspace'
+
+# Schemes
+TESTS_IOS = 'Tests iOS'
+TESTS_OSX = 'Tests OSX'
+ENCRYPTION_IOS = 'Encryption Tests'
+ENCRYPTION_OSX = 'Encryption Tests OSX'
+REPLICATION_ACCEPTANCE_IOS = 'RA_Tests'
+REPLICATION_ACCEPTANCE_OSX = 'RA_Tests_OSX'
+REPLICATION_ACCEPTANCE_ENCRYPTED_IOS = 'RA_EncryptionTests'
+REPLICATION_ACCEPTANCE_ENCRYPTED_OSX = 'RA_EncryptionTests_OSX'
+
+# Destinations
+IPHONE_DEST = 'platform=iOS Simulator,OS=latest,name=iPhone 4S'
+OSX_DEST = 'platform=OS X'
+
+# Runs `build` target for workspace/scheme/destination
+def run_build(workspace, scheme, destination)
```

Comment:
OK, I see. In that case you can remove the message here: https://github.com/cloudant/CDTDatastore/pull/203/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR119

I would be in favour of making the failure messages red. Most tools can interpret the terminal escapes correctly and they don't harm readability too much for those things which can't interpret them.